Docker_run_cuda

```diff
@@ -0,0 +1,32 @@
+ARG BUILD_IMAGE_NAME
+ARG CUDA_IMAGE_NAME=nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+FROM ${BUILD_IMAGE_NAME} AS builder
+FROM ${CUDA_IMAGE_NAME}
+
+ARG PYVER=2.7
+ENV PYVER=${PYVER}
+ARG PYV=27
+ENV PYV=${PYV}
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        curl \
+        git \
+        python$PYVER \
+        python$PYVER-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV PYTHONIOENCODING utf-8
+ENV LC_ALL C.UTF-8
+RUN rm -f /usr/bin/python && \
```

Comment:
I'm also wondering, are these `rm`'s necessary? 
I've checked and the parent docker image doesn't contain any python version, so the only version there will exist is the version we install with `apt-get`
```$ nvidia-docker run -it nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
root@6ab9053a014c:/# python
bash: python: command not found
root@6ab9053a014c:/# ls /usr/bin/pyth*
ls: cannot access '/usr/bin/pyth*': No such file or directory
```

Reply:
Done